Readme.md

```diff
@@ -14,7 +14,7 @@ This is a **Task Management Application** built with a **MERN stack**. The front
 
 ## 📜 Project Overview
 
-_A task management app, where users can organise their personal and team-based tasks, track progress, and share task updates. Users can manage their task boards with features to create, edit, assign, and delete tasks for themselves and other members._
+_A task management app, where users can organize their personal and team-based tasks, track progress, and share task updates. Users can manage their task boards with features to create, edit, assign, and delete tasks for themselves and other members._
```

Comment:
`organize` is the preferred spelling in American English, while `organise` is the preferred spelling in British English